ibis/tests/all/test_geospatial.py

```diff
@@ -103,57 +156,63 @@ def test_get_point(backend, geo, expr_fn, expected):
 
 @pytest.mark.parametrize(('arg', 'expected'), [(polygon_0, [1.98] * 5)])
 @pytest.mark.only_on_backends(all_db_geo_supported)
+@pytest.mark.xfail_unsupported
 def test_area(backend, geo, arg, expected):
     """Testing for geo spatial area operation."""
-    expr = geo[geo.id, arg.area().name('tmp')]
+    geo_id = geo.id if hasattr(geo, 'id') else geo.index
```

Comment:
Just use `geo_id = getattr(geo, 'id', geo.index)` here.

Reply:
that looks much better, thanks. I always forget the default value! thanks!

Reply:
just for record, the change should be: `geo_id = getattr(geo, 'id', 'index')`

Reply:
also it doesn't work ... I will check how to fix that.

Reply:
In the PR #1872 I fixed this issue so it will be just `id` for mapd/omniscidb and postgis